pyanaconda/core/util.py

```diff
@@ -666,19 +666,33 @@ def service_running(service):
     return ret == 0
 
 
-def enable_service(service):
```

Comment:
Please fix typo `moth` in the first commit message.

Reply:
Fixed! GitHub really should have a better mechanism for commit message enhancement suggestions.